.github/workflows/ci.yml

```
@@ -289,7 +289,7 @@ jobs:
       runs-on-as-json-public: ${{ needs.build-info.outputs.runs-on-as-json-public }}
       python-versions-list-as-string: ${{ needs.build-info.outputs.python-versions-list-as-string }}
       # generate no providers constraints only in canary builds - they take quite some time to generate
-      # they are not needed for regular builds, they are only needed to update constraints in canaries
+      # they are not needed for regular builds, they are only needed to upe constraints in canaries
```

```suggestion
      # they are not needed for regular builds, they are only needed to update constraints in canaries
```
unintentional change?